app/models/info_request.rb

```diff
@@ -1453,6 +1453,8 @@ def self.find_in_state(state)
   end
 
   def move_to_public_body(destination_public_body, opts = {})
+    return unless destination_public_body
+
```

Comment:
oddly, the expected "fail" condition is `nil` - there are tests that deliberately pass `nil` as the first argument

if it waits to return, the new functionality in the patch triggers an error by trying to call methods on `Nil` rather than dropping through with a null response